main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-        1) paddle shifters
-        2) 'cruise control' controls
-        3) horn
-        4) volume control
-    Name 2 things you can do with the:   Steering Wheel
-        1) adjust cruise control settings.
-        2) change gear
-        
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1) speedometer
-        2) check engine light
-        3) tachometer
-        4) fuel gauge
-    Name 3 things you can do with the:   Instrument Cluster
-        1) check current speed
-        2) check current engine revs/min
-        3) check fuel level
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1) navigator side temperature control
-        2) driver side temperature control 
-        3) fan speed control
-    Name 3 things you can do with the:   Environment Controls
-        1) adjust fan speed
-        2) adjust driver side temperature
-        3) adjust navigator side temperature
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) "now playing" track
-        2) navigation 
-        3) ambient temperature display
-    Name 3 things you can do with the:   Infotainment System
-        1) select input device
-        2) start navigation 
-        3) start/pause playback
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) seat covers
-        2) ventilation
-        3) seat warmers
-    Name 2 things you can do with the:   Seat
-        1) heat up the seat
-        2) change seat position
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) CPU
+5 properties:
+    1) number of cores
+    2) clock speed
+    3) L1 cache
+    4) L2 cache
+    5) L3 cache
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise

Reply:
Hmm.. My thought process was that these could be represented by an integer, eg. int l1Cache = 64. I can still change this if you'd like, but does that make sense?